nixpkgs-src.json

```diff
@@ -1,6 +1,6 @@
 {
-    "owner":  "NixOS",
+    "owner":  "input-output-hk",
```

Comment:
This will break Hydra, since it doesn't account for a different owner in `jobsets/default.nix`

Reply:
This is a temporary stub for testing -- will drop that change soon.